.github/workflows/sdk-regression.yml

```diff
@@ -42,10 +42,10 @@ jobs:
             const sha = '${{ steps.comment-branch.outputs.head_sha }}'
             const state = 'pending';
             const target_url = '${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}'
-            const context = 'SDK Regression ${{ matrix.repo }}'
+            const check_name = 'SDK Regression ${{ matrix.repo }}'
```

Comment:
question: does this needs to be `'regression ${{ matrix.repo }}'` ? from the actions tab we're having container name as this.

Reply:
Nope can be anything